exchanges/credentials.go

```diff
@@ -110,12 +110,12 @@ func (b *Base) GetDefaultCredentials() *account.Credentials {
 
 // GetCredentials checks and validates current credentials, context credentials
 // override default credentials, if no credentials found, will return an error.
-func (b *Base) GetCredentials(ctx context.Context) (*account.Credentials, error) {
-	value := ctx.Value(account.ContextCredentialsFlag)
+func (b *Base) GetCredentials(ctx context.Context) (*accounts.Credentials, error) {
+	value := ctx.Value(accounts.ContextCredentialsFlag)
 	if value != nil {
-		ctxCredStore, ok := value.(*account.ContextCredentialsStore)
+		ctxCredStore, ok := value.(*accounts.ContextCredentialsStore)
```

Comment:
suggestion: could rm this check as it should be set by the deploy func correctly